README.md

```diff
@@ -16,7 +16,7 @@ This will provide you with a `client_id` and a `client_secret`.
 Let's say you wanted to make a "Log In with Patreon" button.
 You've read through [the directions](https://www.patreon.com/platform/documentation/oauth),
 and are trying to implement "Step 2: Handling the OAuth Redirect" with your server.
-The user will be arriving at one of your pages *after* you have sent them to [the authorize page](www.patreon.com/oauth2/authorize) for step 1,
+The user will be arriving at one of your pages *after* you have sent them to the authorize page (at https://www.patreon.com/oauth2/authorize) for step 1,
```

Comment:
Actually clicking on that link is useless.

Reply:
Good point